qdrant-landing/content/documentation/tutorials/semantic-search-beginners.md

```diff
@@ -0,0 +1,156 @@
+---
+title: Semantic Search for Beginners
+weight: 14
+---
+
+# Semantic Search for Beginners
+
+| Time: 5 - 15 min | Level: Beginner | Output: [GitHub]() | [![Open In Colab](https://colab.research.google.com/assets/colab-badge.svg)]()   |
+| --- | ----------- | ----------- |----------- |
+
+<center><iframe src="https://www.youtube.com/embed/hFc7xv1tsos" width="900" height="500" title="Semantic Search for Beginners" frameborder="0" allowfullscreen></iframe></center>
+
+If you are new to vector databases, this tutorial is for you. In 5 minutes you will build a semantic search engine for science fiction books. After you set it up, you will ask the engine about an impending alien threat. Your creation will recommend books as preparation for a potential space attack.
+
+Before you begin, you need to have a [recent version of Python](https://www.python.org/downloads/) installed. If you don't know how to run this code in a virtual environment, follow [this tutorial](https://towardsdatascience.com/creating-and-using-virtual-environment-on-jupyter-notebook-with-python-db3f5afdd56a) first.
+
+## 1. Installation
+
+You need to process your data so that the search engine can work with it. The [Sentence Transformers] framework gives you access to common [Large Language Models] that turn raw data into embeddings.
+```python
+pip install -U sentence-transformers
+```
+
+Once encoded, this data needs to be kept somewhere. Qdrant lets you store data as embeddings. You can also use Qdrant to run search queries against this data. This means that you can ask the engine to give you relevant answers that go way beyond keyword matching.
+```python
+pip install qdrant-client
+```
+
+### Import the models 
+
+Once the two main frameworks are defined, you need to specify the exact models this engine will use. 
+```python
+from qdrant_client import models, QdrantClient
+from sentence_transformers import SentenceTransformer
+```
+
+The [Sentence Transformers] framework contains many Large Language Models. However, [all-MiniLM-L6-v2] is the fastest encoder for this tutorial.
+```python
+encoder = SentenceTransformer('all-MiniLM-L6-v2') 
+```
+
+## 2. Add the dataset
+
+[all-MiniLM-L6-v2] will encode the data you provide. Here you will list all the science fiction books in your library. Each book has metadata, a name, author, publication year and a short description. 
```

Comment:
The same as above:
`[all-MiniLM-L6-v2]` -> `[all-MiniLM-L6-v2](https://huggingface.co/sentence-transformers/all-MiniLM-L6-v2)`